README.md

```diff
@@ -91,13 +91,13 @@ specific RuboCop release.**
 
 **RuboCop**'s installation is pretty standard:
 
-```
+```sh
```

Comment:
Isn't `sh` for highlighting `sh` scripts? To my knowledge there's nothing for our exact situation (although I've seen many people use `sh` and `bash` for this).